app/router.js

```diff
@@ -36,6 +36,7 @@ Router.map(function() {
     this.route('organization-invites', function() {
       this.route('new');
     });
+    this.route('projects', function() {});
```

Comment:
I think the `function` is unnecessary here.

Reply:
I've been meaning to comment on that.

Without the `function`, there's no `projects.index` route being autogenerated, so we only get the `projects` route, meaning the file structure is not consistent with github events and organization invites.

Really, that would be fine, but my assumption is, we'll soon have other subroutes of projects anyway, so leaving this here might save us some time.

I could

- leave this here
- leave this here and add a comment to explain why it's here
- remove this and 
  - move `routes/admin/projects/index.js` -> `/routes/admin/project.js/`
  - move `controllers/admin/projects/index.js` -> `/controllers/admin/project.js/`
  - move `tests/pages/admin/projects/index.js` -> `tests/pages/admin/projects.js`

Reply:
I think fine leaving it for now.